CRITICAL - 2024-01-25 10:39:27 --> Undefined variable $totalTime
in SYSTEMPATH\Debug\Toolbar\Views\toolbar.tpl.php on line 46.
 1 SYSTEMPATH\Debug\Toolbar\Views\toolbar.tpl.php(46): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $totalTime', 'C:\\xampp\\htdocs\\DWES\\wallapop\\system\\Debug\\Toolbar\\Views\\toolbar.tpl.php', 46)
 2 SYSTEMPATH\Debug\Toolbar.php(513): include('C:\\xampp\\htdocs\\DWES\\wallapop\\system\\Debug\\Toolbar\\Views\\toolbar.tpl.php')
 3 SYSTEMPATH\Debug\Toolbar.php(476): CodeIgniter\Debug\Toolbar->format([...], 'html')
 4 APPPATH\Config\Events.php(47): CodeIgniter\Debug\Toolbar->respond()
 5 SYSTEMPATH\Events\Events.php(154): CodeIgniter\Events\Events::Config\{closure}()
 6 SYSTEMPATH\CodeIgniter.php(442): CodeIgniter\Events\Events::trigger('pre_system')
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-25 13:12:36 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\DWES\wallapop\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\DWES\wallapop\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'prueba_ci', 8889, '', 0)
#1 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ar...', Array, false)
#4 C:\xampp\htdocs\DWES\wallapop\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\DWES\wallapop\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\DWES\wallapop\app\Controllers\Articulos.php(36): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(942): App\Controllers\Articulos->guardar()
#10 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Articulos))
#11 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\DWES\wallapop\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\DWES\wallapop\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ar...', Array, false)
#3 C:\xampp\htdocs\DWES\wallapop\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\DWES\wallapop\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\DWES\wallapop\app\Controllers\Articulos.php(36): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(942): App\Controllers\Articulos->guardar()
#9 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Articulos))
#10 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\DWES\wallapop\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-25 13:12:36 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `articulos` (`nombre`, `marca`, `precio`, `imagen`) VALUES (:nombre:, :marca:, :precio:, :imagen:)', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\Articulos.php(36): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Articulos->guardar()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Articulos))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-25 13:13:13 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'prueba_ci' in C:\xampp\htdocs\DWES\wallapop\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\DWES\wallapop\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'prueba_ci', 3306, '', 0)
#1 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ar...', Array, false)
#4 C:\xampp\htdocs\DWES\wallapop\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\DWES\wallapop\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\DWES\wallapop\app\Controllers\Articulos.php(36): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(942): App\Controllers\Articulos->guardar()
#10 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Articulos))
#11 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\DWES\wallapop\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'prueba_ci' in C:\xampp\htdocs\DWES\wallapop\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\DWES\wallapop\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ar...', Array, false)
#3 C:\xampp\htdocs\DWES\wallapop\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\DWES\wallapop\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\DWES\wallapop\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\DWES\wallapop\app\Controllers\Articulos.php(36): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(942): App\Controllers\Articulos->guardar()
#9 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Articulos))
#10 C:\xampp\htdocs\DWES\wallapop\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\DWES\wallapop\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-25 13:13:13 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'prueba_ci'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `articulos` (`nombre`, `marca`, `precio`, `imagen`) VALUES (:nombre:, :marca:, :precio:, :imagen:)', [...], false)
 3 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\Articulos.php(36): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Articulos->guardar()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Articulos))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-25 13:13:27 --> Invalid file: "alta_articulo.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('alta_articulo.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('alta_articulo', [], true)
 3 APPPATH\Controllers\Articulos.php(40): view('alta_articulo')
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Articulos->guardar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Articulos))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
